src/braket/circuits/circuit.py

```diff
@@ -431,7 +431,7 @@ def add_circuit(
                 Default = `None`.
             target_mapping (dictionary[int or Qubit, int or Qubit], optional): A dictionary of
                 qubit mappings to apply to the qubits of `circuit.instructions`. Key is the qubit
-                to map, and the value is what to change it to. Default = `{}`.
+                to map, and the value is what to change it to. Default = `None`.
```

Comment:
Do these require updates in the implementation or are they already handled by the methods being called here ?

Reply:
No functional changes; where the default `{}` used to be used, I've changed the usage to from `func(mapping,)` to `func(mapping or {})`.